lang/cs/Org.Apache.REEF.Client/YARN/IJobResourceUploader.cs

```diff
@@ -28,6 +29,6 @@ public interface IJobResourceUploader
         /// <param name="driverLocalFolderPath">Local folder where REEF application resources are staged</param>
         /// <param name="jobSubmissionDirectory">Remote directory path where we will upload resources</param>
         /// <returns>Path, modification time and size of uploaded file as JobResource</returns>
-        JobResource UploadJobResource(string driverLocalFolderPath, string jobSubmissionDirectory);
+        ICollection<JobResource> UploadJobResource(string driverLocalFolderPath, string jobSubmissionDirectory);
```

Comment:
Why are we changing this to collection?
In case we have to the consider using IReadOnlyCollection instead.